packages/alfa-rules/src/sia-r28/rule.ts

```diff
@@ -40,10 +38,7 @@ export default Rule.Atomic.of<Page, Element>({
         return {
           1: expectation(
             test(
-              hasAccessibleName(
-                device,
-                (accessibleName) => accessibleName.value !== "Submit Query"
-              ),
+              hasAccessibleName(device, (name) => name.source.length !== 0),
```

Comment:
Let's help our future selves.
```suggestion
              // Rejecting default name.
              hasAccessibleName(device, (name) => name.source.length !== 0),
```